proposed/2022/package-scoring-v1.md

```diff
@@ -0,0 +1,219 @@
+# Package Scoring v1
+
+- [Jon Douglas](https://github.com/JonDouglas)
+- Start Date (2022-05-16)
+- [dotnet/designs#216](https://github.com/dotnet/designs/pull/216)
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+Having packages of high quality is crucial to the .NET ecosystem. As a first step to increase awareness of the characteristics that make a package high quality, we are proposing the first iteration of an effort known as package scoring.
+
+This proposal and initial iteration will focus on common additions to a NuGet package that benefits the entirity of the .NET developer ecosystem. For reference, majority of developers we survey and chat with talk about 5 key characteristics that they deem a high quality or "healthy" package:
+
+A high quality or "healthy" package is one that follows the following characteristics:
+
+- It is actively maintained. Either with recent commits or an annual update/notice that the package is up-to-date.
+- It is documented. It provides enough documentation to install, get started, and has public API documentation on it's members.
+- It has a way to report bugs. It provides a centralized location or repository in which issues are regularly triaged & resolved in future releases.
+- It resolves security flaws quickly. It fixes known vulnerabilities & releases an unaffected release quickly.
+- It is not deprecated. It is in an active state meeting all the criteria above.
+
+This proposal introduces the first iteration of package scoring. While it derives from the original proposal [last year on this topic](https://github.com/dotnet/designs/pull/216), it provides the minimum requirements for a v1.0 of implementing this on NuGet.org.
```

Comment:
I assume "on NuGet.org" means that Visual Studio and dotnet CLI are out of scope. Therefore, there are also no proposed changes to the NuGet protocol (that clients use to talk to servers).

Is it worth making any of this more explicit?